examples/pandapipes/input/timeseries_constant.csv

```diff
@@ -0,0 +1,46 @@
+DateTime,demand_1
+2013-05-19 22:00:00,50000000.0
+2013-05-19 23:00:00,44444444.0
+2013-05-20 00:00:00,38888889.0
+2013-05-20 01:00:00,33333333.0
+2013-05-20 02:00:00,27777778.0
+2013-05-20 03:00:00,22222222.0
+2013-05-20 04:00:00,16666667.0
+2013-05-20 05:00:00,11111111.0
+2013-05-20 06:00:00,5555556.0
+2013-05-20 07:00:00,1.0
+2013-05-20 08:00:00,50000000.0
```

Comment:
Can we not simply delete everything from line 12 onwards?

Reply:
Yes. Done